core/vm/interpreter.go

```diff
@@ -118,15 +118,15 @@ func NewEVMInterpreter(evm *EVM, cfg Config) *EVMInterpreter {
 		jt = &frontierInstructionSet
 	}
 	if len(cfg.ExtraEips) > 0 {
-		jtCopy := *jt
+		jtCopy := jt.Clone()
```

Comment:
Why is this needed? Is jump table populated further from some other threads?

Reply:
Not really, hovewer, jump tables are global variables and a bunch EIPs can be applied over this variables, changing them.

Reply:
I have a concern that this cloning will be happening at every transaction. We actually remove some kind of cloning like this before because it was taking a large percentage of time in the profiling

Reply:
Will profile before and after the changes.

Reply:
I would not invest much into it. We definitely do not want to do this cloning, it has been shown in the past to be expensive. Let's rethink this fix as a whole, we can find a much better solution

Reply:
Ok, I removed cloning. Could you take a look at it?

Reply:
Now I looked at it more closely. I think the reason you were  `ExtraEips` is currently only used in tests and in the `t8n` tool, and not in the actual instruction sets in production. In production those `jt` are are immutable once constructed. So I think the original data race probably occurred because tests were running in parallel. Now I see that your cloning solution, or removing the `t.Parallel()` would both do the trick. I would prefer just remove `t.Parallel()` to be honest, and remove rest of the code, I don't think it adds anything

Reply:
Done